### PR TITLE
Fix allignment on layout on right panel

### PR DESCRIPTION
**Pull request by harikrishna-patnala**
### Description

This is a followup for the PR https://github.com/apache/cloudstack/pull/7434, here we are making some more improvements

from
image

to
image

Along the with search box allignment, fixed the font sizes and check box sizes

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by harikrishna-patnala** [comment]
@blueorangutan ui

**Review by borisstoyanov — APPROVED** [review]
LGTM
Screenshot 2023-10-16 at 14 25 04

**Comment by rohityadavcloud** [comment]
LGTM but I think the button makes it awkward - should button be similarly aligned by height?
Screenshot 2023-10-19 at 4 57 36 PM

**Review by rohityadavcloud — APPROVED** [review]

File: ui/src/views/AutogenView.vue [hunk 1]
```diff
@@ -54,7 +54,8 @@
                     v-if="!dataView && filters && filters.length > 0"
                     :placeholder="$t('label.filterby')"
                     :value="filterValue"
-                    style="min-width: 120px; margin-left: 10px; margin-top: -4px"
+                    style="min-width: 100px; margin-left: 10px; margin-top: -4px"
+                    size=small
```
Comment (DaanHoogland) [inline ui/src/views/AutogenView.vue:58]: I've seen console errors on "large" and "small" in the past. Where are thos defined, and are we sure this is not an issue here?

**Review by DaanHoogland — APPROVED** [review]
looks good generally, one question

**Comment by andrijapanicsb** [comment]
> LGTM but I think the button makes it awkward - should button be similarly aligned by height? Screenshot 2023-10-19 at 4 57 36 PM

That is the whole point, both Project/Metrics are shorter (not the same height) than the "Refresh" button and "Active" dropdown), while the main BLUE buttons (Add instance, Add network, Add VPC, etc) - are higher than the rest - which we might want to make same heigh as the white buttons?

File: ui/src/views/AutogenView.vue (hunk 1 above)
Comment (harikrishna-patnala) [reply ui/src/views/AutogenView.vue:58]: I dont see any errors now @DaanHoogland, probably due to ant3 upgrade.

**Comment by harikrishna-patnala** [comment]
@blueorangutan ui

**Comment by harikrishna-patnala** [comment]
@blueorangutan ui

**Review by andrijapanicsb — APPROVED** [review]
LGTM, after visual inspection

**Pull request merged**